analysers/trials/constructDuration.py

```diff
@@ -0,0 +1,98 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from datetime import timedelta
+
+import numpy as np
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+def createQuery(sc, dataRDD, experimentID, trialID):
+    from commons import computeMode, computeMetrics
+    
+    queries = []
+    
+    modeRDD = dataRDD.filter(lambda r: r['duration'] != None) \
```

Comment:
@VincenzoFerme I use Spark for the mode, unlike the others that use numpy (numpy gives you a single mode value, I want all of them)